app/controllers/employer/base_controller.rb

```diff
@@ -1,9 +1,9 @@
 class Employer::BaseController < ApplicationController
```

Comment:
@jmejia: makes sense. But if we just have a method to check for a user role on the base controller:
 ``` def require_employer
    unless @current_user.employer?
      render file: '/public/404', status => 404, :layout => true
    end
  end```
I'm assuming that we're already taking care of authorization with services. Is that right?